File: tau WinAPP driver/CalculatorPython/notepad/notepad_PO.py
class notepad_PO():
    def __init__(self,wd):
        self.driver = wd
    

    def textArea(self):
        return self.driver.find_element_by_name("Text Editor")

    def dialogDontSave(self):
        return self.driver.find_element_by_name("Don't Save")

    def dialogCancel(self):
        return self.driver.find_element_by_name("Cancel")

File: tau WinAPP driver/CalculatorJava/src/main/java/Notepad_PO.java
import io.appium.java_client.windows.WindowsDriver;
import org.openqa.selenium.WebElement;

public class Notepad_PO {
    private WindowsDriver driver = null;
    public Notepad_PO(WindowsDriver wd){
        driver = wd;
    }

    public WebElement Minimize(){
        return driver.findElementByName("Minimize");
    }
    public WebElement Maximize(){
        return driver.findElementByName("Maximize");
    }
    public WebElement CLose(){
        return driver.findElementByName("CLose");
    }

    public WebElement MenuFile(){
        return driver.findElementByName("File");
    }

    public WebElement TextArea(){
        return driver.findElementByName("Text Editor");
    }

    public WebElement DialogSave(){
        return driver.findElementByName("Save");
    }
    public WebElement DialogDontSave(){
        return driver.findElementByName("Don't Save");
    }

    public WebElement DialogCancel(){
        return driver.findElementByName("Cancel");
    }


}
